config/config.go

```diff
@@ -83,6 +83,9 @@ func init() {
 	cfg.SetDefault("analyzer.storage.bulk_insert", 100)
 	cfg.SetDefault("analyzer.storage.bulk_insert_deadline", 5)
 	cfg.SetDefault("analyzer.storage.max_flow_buffer_size", 100000)
+	cfg.SetDefault("analyzer.storage.index_entries_limit", -1)
+	cfg.SetDefault("analyzer.storage.index_age_limit", -1)
+	cfg.SetDefault("analyzer.storage.indices_to_keep", -1)
 	cfg.SetDefault("analyzer.topology.probes", []string{})
```

Comment:
nit: better to order these alphabetically:
- index_age_limit
- index_entries_limit
- indices_to_keep